01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,41 @@
+//getting the built in file system module
+const {writeFile, readFile } = require("fs").promises;
+
+//3 lines
+const content = `This is line 1, 
+This is line dos,
+Lastly line 3.`;
+
+//async writer function
+async function writer(){
```

Comment:
For your question: "Why can’t we call async in our main line of code?"

By default, you can't use the await keyword at the top level in older versions of Node. That’s because await can only be used inside an async function. If you try to use await at the top level, it will result in an error. So in order to use it its inside the async scope.